src/screens/LoginScreen/LoginScreen.tsx

```diff
@@ -55,6 +55,7 @@ export default () => {
 				autoCorrect={false}
```

Comment:
Revert this file

The PR changes should be constrained to the [Trello card](https://trello.com/c/1l1Q7j24). So no need to update the login screen. Another PR request will handle the unique constraints of the login form fields.